cmd/integration_test/build/buildspecs/conformance-eks-a-cli.yml

```
@@ -13,6 +13,9 @@ env:
   secrets-manager:
     EKSA_VSPHERE_USERNAME: "vsphere_ci_beta_connection:vsphere_username"
     EKSA_VSPHERE_PASSWORD: "vsphere_ci_beta_connection:vsphere_password"
+    EKSA_AWS_ACCESS_KEY_ID: "packages_ci_beta:aws_access_key_id"
+    EKSA_AWS_SECRET_ACCESS_KEY: "packages_ci_beta:aws_secret_access_key_id"
+    EKSA_AWS_REGION: "us-west-2"
```

I believe this should be, 
```suggestion
    EKSA_AWS_REGION: "packages_ci_beta:aws_region"
```
If the intent is not to get this from secret manager and just hardcode, then it needs to be moved up to the variables section instead of secret manager section.

---

ok will change and add that to secret manager